db/python/layers/seqr.py

```diff
@@ -43,6 +43,22 @@
 
 SEQUENCING_GROUPS_TO_IGNORE = {22735, 22739}
 
+DATASET_TYPE_SNV_INDEL_CALLS = 'SNV_INDEL'
+DATASET_TYPE_SV_CALLS = 'SV'
+DATASET_TYPE_MITO_CALLS = 'MITO'
+
+ES_INDEX_DATASET_TYPES = {
```

Comment:
Can you add some commentary about what definitions go where (even consider more descriptive variable names). IE: of these is seqr, one is production-pipelines, and one is UI driven if I'm understanding correctly. If it's one that metamist controls, then our favour is complete upper case